README.md

```diff
@@ -36,15 +36,18 @@ Alternatively you can specify a filter for the services you want updated using t
 
 You can enable private registry authentication by setting the `WITH_REGISTRY_AUTH` variable.
 
+`NO_VERBOSE` variable can disable info messages like "Trying to update ..." and "No updates to service ...".
+
 Example:
 
     docker service create --name shepherd \
                         --constraint "node.role==manager" \
                         --env SLEEP_TIME="5m" \
                         --env BLACKLIST_SERVICES="shepherd my-other-service" \
                         --env WITH_REGISTRY_AUTH="true" \
-                        --env FILTER_SERVICES="label=com.mydomain.autodeploy"
-                        --mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock,ro \
+                        --env FILTER_SERVICES="label=com.mydomain.autodeploy" \
+                        --env NO_VERBOSE="true" \
+                        --mount type=bind,source=/var/run/docker.sock,target=/var/run/docker.sock \
```

Comment:
The `,ro` suffix can be kept. (It still allows write access to the socket, just preventing it from being overwritten with something else.)